src/fe-fuzz/module-formats.c

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
I wonder if we could #include the other formats instead of having top copy*paste?